### PR TITLE
feat(sms-limiting): ui changes for modal

### PR DESCRIPTION
**Pull request by seaerchin**
## Description
This changes the UI to be more polished.

This takes place on 3 fronts: 
1. adding a quota beneath the toggle so that users are able to see their sms counts. 
2. changing the sms counts modal button from red -> blue.
3. updating the sms counts modal description.

## Screenshots
**Quota**
Screenshot 2021-08-06 at 12 16 54 PM

**Modal**
Screenshot 2021-08-06 at 12 17 39 PM

### CODE REVIEW

File: src/public/modules/forms/admin/directiveViews/configure-mobile.client.view.html [hunk 1]
```diff
@@ -1,15 +1,22 @@
 <div class="row">
-  <div class="toggle-option">
+  <div class="toggle-option label-bottom">
     <div
       ng-class="{'toggle-option--disabled' : field.hasAdminExceededSmsLimit || field.hasRetrievalError}"
-      class="col-xs-8 label-custom label-medium label-bottom"
+      class="col-xs-8 justify-content-start"
```
Comment (karrui) [inline src/public/modules/forms/admin/directiveViews/configure-mobile.client.view.html:5]: Can you double check if `justify-content-start` does anything? We are still using bootstrap 3, I don't see any documentation on this css

**Review by karrui — APPROVED** [review]
small comment, otherwise lgtm

File: src/public/modules/forms/admin/directiveViews/configure-mobile.client.view.html (hunk 1 above)
Comment (seaerchin) [reply src/public/modules/forms/admin/directiveViews/configure-mobile.client.view.html:5]: I referenced the newer version of bootstrap [here](https://getbootstrap.com/docs/4.0/layout/grid/#horizontal-alignment) where it aligned horizontally as I wasn't aware of the exact version of bootstrap used. 

Will double check if this does anything

Comment (karrui) [reply src/public/modules/forms/admin/directiveViews/configure-mobile.client.view.html:5]: The current version of bootstrap being used can be found in our package.json

Comment (seaerchin) [reply src/public/modules/forms/admin/directiveViews/configure-mobile.client.view.html:5]: checked; this is extra, will remove

**Pull request merged**